client/game.js

```diff
@@ -64,10 +64,11 @@ const app = new Vue({
       );
     },
     canMoveTo(x, y) {
-      return (
-        distance(this.me.position, { x, y }) ==
-          this.config.Range.MoveDistance && this.me.health > 0
-      );
+      //checks if you can move forwards
+      straight = distance(this.me.position, { x, y }) == this.config.Range.MoveDistance
+      //checks if you can move diagonally
+      diagonal = (distance(this.me.position, { x, y })).toFixed(3) == (sqrt(2) * this.config.Range.MoveDistance).toFixed(3)
```

Comment:
1. Need to declare the variable `diagonal` before using it.
2. `sqrt` is a function on the `Math` package

```suggestion
      const diagonal = (distance(this.me.position, { x, y })).toFixed(3) == (Math.sqrt(2) * this.config.Range.MoveDistance).toFixed(3)
```